Formula/ctail.rb

```diff
@@ -12,15 +13,21 @@ class Ctail < Formula
     sha256 "dfab40d65950327c679bde97a335779526c58e99f5679f32f95e517a7249e332" => :mountain_lion
   end
 
-  conflicts_with "byobu", :because => "both install `ctail` binaries"
-
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
+  depends_on "apr" => :build
+  depends_on "apr-util" => :build
+
+  conflicts_with "byobu", :because => "both install `ctail` binaries"
 
   def install
-    system "./configure", "--prefix=#{prefix}", "--disable-debug"
+    system "./configure", "--prefix=#{prefix}", "--disable-debug", "--with-apr=#{Formula["apr"].bin}", "--with-apr-util=#{Formula["apr-util"].bin}"
     system "make", "LIBTOOL=glibtool --tag=CC"
     system "make", "install"
   end
+
+  test do
+    system "ctail", "-h"
```

Comment:
Test is failing: https://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/8708/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/test_ctail/

Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

Finally, regardless it should be `system "#{bin}/ctail", "-h"`